daily-coding-challenges/26-05-2020/findGreatestNumInArray.js

```diff
@@ -4,6 +4,7 @@
 */
 const findGreatestNumInArray = (arr) => {
 	/* Your implementation goes here */
+	if (arr.legnth ==1) {return arr[0]}
```

Comment:
@iamfortune, okay this is good. But I see a typo here, can you correct this from `legnth` to `length`?

Reply:
done this, but somehow i couldn't seem to get the edge case for null and undefined to work!>/
will gladly accept help on this front